app/Actions/Concerns/InteractsWithGitHub.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App\Actions\Concerns;
+
+use App\Configuration\LamboConfiguration;
+use App\LamboException;
+
+trait InteractsWithGitHub
+{
+
+    protected function shouldCreateRepository(): bool
+    {
+        return $this->gitHubInitializationRequested() && $this->gitHubToolingInstalled();
+    }
+
+    protected function gitHubInitializationRequested(): bool
+    {
+        return config('lambo.store.' . LamboConfiguration::INITIALIZE_GITHUB) === true;
+    }
+
+    protected function getDescription(): string
+    {
+        $description = config('lambo.store.' . LamboConfiguration::GITHUB_DESCRIPTION);
+        if (is_null($description)) {
+            return '';
+        }
+
+        return sprintf(' --description="%s"', $description);
+    }
+
+    protected function getHomepage(): string
+    {
+        $homepage = config('lambo.store.' . LamboConfiguration::GITHUB_HOMEPAGE);
+        if (is_null($homepage)) {
+            return '';
+        }
+
+        return sprintf(' --homepage="%s"', $homepage);
+    }
+
+    /**
+     * @throws LamboException
+     */
+    protected function getCommand(): string
+    {
+        if ($this->hubInstalled()) {
```

Comment:
Should we maybe default to `gh` if installed first, since that's the official one?

Reply:
@mattstauffer when I initially added hub I remember prioritizing hub for a reason—a reason I can't remember now 😂..
In any case I've made `gh` the default, run the tests and tested manually. Everything seems to be working. I did do a big refactor mid-development (introduced the InteractsWithGitHub trait) and maybe that fixed it 🤷🏻‍♂️.